toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -231,31 +231,3 @@ func systemdFormatCorruptionOption(corruptionOption imagecustomizerapi.Corruptio
 		return "", fmt.Errorf("invalid corruptionOption provided (%s)", string(corruptionOption))
 	}
 }
-
-// findFreeNBDDevice finds the first available NBD device.
-func findFreeNBDDevice() (string, error) {
```

Comment:
curious, why were we using nbd devices for the verity flow?

Reply:
Lanze claimed using loopback resulted in the partition being changed, which invalidates the hash. When writing the verity test, I didn't want to deal with the `nbd` dependency for the test. So, I tested to see if I could observe the issue. And I could not. Hence, I fixed it.